sotodlib/tod_ops/t2pleakage.py

```diff
@@ -4,17 +4,17 @@
 from sotodlib import core
 from sotodlib.tod_ops import filters, apodize
 from sotodlib.tod_ops.fft_ops import calc_psd, calc_wn
-from scipy.odr import ODR, Model, RealData
+from lmfit import Model
+    
+def leakage_model(dT, AQ, AU, lamQ, lamU):
+    return AQ + lamQ * dT + 1.j * (AU + lamU * dT)
 
-def get_t2p_coeffs(aman, 
```

Comment:
I would like to leave in the possibility to run the old method (which doesn't include a joint fit or AQ/AU) as that wraps different coeff names into the metadata that's saved in the preprocess database so this will make it impossible to load data from the old databases.